app/policies/client_policy.rb

```diff
@@ -2,19 +2,23 @@ class ClientPolicy < ApplicationPolicy
   class Scope < ApplicationScope
     def resolve
       return all if superadmin?
-      resolve_owner if social_worker?
```

Comment:
If I got it right yesterday at the standup, we keep the social workers with their current right.